tests/projects/test_projects_cli.py

```diff
@@ -1,25 +1,29 @@
 import pytest
 
-import mlflow
 from mlflow import cli
-from mlflow.utils.file_utils import TempDir
-from tests.integration.utils import invoke_cli_runner, update_temp_env
-from tests.projects.utils import TEST_PROJECT_DIR, GIT_PROJECT_URI
+from tests.integration.utils import invoke_cli_runner
+from tests.projects.utils import TEST_PROJECT_DIR, GIT_PROJECT_URI, SSH_PROJECT_URI
+from tests.projects.utils import tracking_uri_mock  # pylint: disable=unused-import
 
 
 @pytest.mark.large
-def test_run_local():
-    with TempDir() as tmp:
-        with update_temp_env({mlflow.tracking._TRACKING_URI_ENV_VAR: tmp.path()}):
-            excitement_arg = 2
-            name = "friend"
-            invoke_cli_runner(cli.run, [TEST_PROJECT_DIR, "-e", "greeter", "-P",
-                                        "greeting=hi", "-P", "name=%s" % name,
-                                        "-P", "excitement=%s" % excitement_arg])
+def test_run_local(tracking_uri_mock):  # pylint: disable=unused-argument
+    excitement_arg = 2
+    name = "friend"
+    invoke_cli_runner(cli.run, [TEST_PROJECT_DIR, "-e", "greeter", "-P",
+                                "greeting=hi", "-P", "name=%s" % name,
+                                "-P", "excitement=%s" % excitement_arg])
 
 
 @pytest.mark.large
-def test_run_git():
-    with TempDir() as tmp:
-        with update_temp_env({mlflow.tracking._TRACKING_URI_ENV_VAR: tmp.path()}):
-            invoke_cli_runner(cli.run, [GIT_PROJECT_URI, "--no-conda", "-P", "alpha=0.5"])
+def test_run_git_https(tracking_uri_mock):  # pylint: disable=unused-argument
+    # Invoke command twice to ensure we set Git state in an isolated manner (e.g. don't attempt to
+    # create a git repo in the same directory twice, etc)
+    invoke_cli_runner(cli.run, [GIT_PROJECT_URI, "-P", "alpha=0.5"])
```

Comment:
nit: could we assert that `GIT_PROJECT_URI` startswith `https`? The name of the parameter does not indicate that it's necessarily https, so I could see a future PR alter it without thinking about this particular test. Similar thing with the git test.